weapons/main.json

```diff
@@ -181,11 +197,239 @@
+      "categories": ["All", "Beam", "UnchargedBeam"]
+    },
+    {
+      "id": 25,
+      "name": "Charge+Ice+Wave+Spazer",
+      "damage": 300,
+      "cooldownFrames": 60,
+      "useRequires": ["Charge", "Ice", "Wave", "Spazer", "PowerBeam"],
+      "situational": false,
+      "hitsGroup": false,
+      "categories": ["All", "Beam"]
+    },
+    {
+      "id": 26,
+      "name": "Ice+Plasma",
+      "damage": 200,
+      "cooldownFrames": 12,
+      "useRequires": ["Ice", "Plasma", "PowerBeam"],
+      "situational": false,
+      "hitsGroup": false,
+      "categories": ["All", "Beam", "UnchargedBeam"],
+      "note": "Special case of extra fast cooldown"
+    },
+    {
+      "id": 27,
+      "name": "Charge+Ice+Plasma",
+      "damage": 600,
+      "cooldownFrames": 60,
+      "useRequires": ["Charge", "Ice", "Plasma", "PowerBeam"],
+      "situational": false,
+      "hitsGroup": false,
+      "categories": ["All", "Beam" ]
+    },
+    {
+      "id": 28,
+      "name": "Wave+Plasma",
+      "damage": 250,
+      "cooldownFrames": 15,
+      "useRequires": ["Wave", "Plasma", "PowerBeam"],
+      "situational": false,
+      "hitsGroup": false,
+      "categories": ["All", "Beam", "UnchargedBeam"]
+    },
+    {
+      "id": 29,
+      "name": "Charge+Wave+Plasma",
+      "damage": 750,
+      "cooldownFrames": 60,
+      "useRequires": ["Charge", "Wave", "Plasma", "PowerBeam"],
+      "situational": false,
+      "hitsGroup": false,
+      "categories": ["All", "Beam" ]
+    },
+    {
+      "id": 30,
+      "name": "Ice+Wave+Plasma",
+      "damage": 300,
+      "cooldownFrames": 15,
+      "useRequires": ["Ice", "Wave", "Plasma", "PowerBeam"],
+      "situational": false,
+      "hitsGroup": false,
+      "categories": ["All", "Beam", "UnchargedBeam"]
+    },
+    {
+      "id": 31,
+      "name": "Charge+Ice+Wave+Plasma",
+      "damage": 900,
+      "cooldownFrames": 60,
+      "useRequires": ["Charge", "Ice", "Wave", "Plasma", "PowerBeam"],
+      "situational": false,
+      "hitsGroup": false,
+      "categories": ["All", "Beam" ]
+    },
+    {
+      "id": 32,
+      "name": "Ice+Wave+Spazer",
+      "damage": 100,
+      "cooldownFrames": 15,
+      "useRequires": ["Ice", "Wave", "Spazer", "PowerBeam"],
+      "situational": false,
+      "hitsGroup": false,
+      "categories": ["All", "Beam", "UnchargedBeam"]
+    },
+    {
+      "id": 33,
+      "name": "Charge+Ice+Wave+Spazer",
+      "damage": 300,
+      "cooldownFrames": 60,
+      "useRequires": ["Charge", "Ice", "Wave", "Spazer", "PowerBeam"],
+      "situational": false,
+      "hitsGroup": false,
+      "categories": ["All", "Beam"]
+    },
+    {
+      "id": 34,
+      "name": "Wave Shield",
+      "damage": 1200,
+      "cooldownFrames": 400,
+      "useRequires": ["Wave", "PowerBomb", "Charge"],
+      "shotRequires": [
+        {"ammo": {
+          "type": "PowerBomb",
+          "count": 1
+        }}
+      ],
+      "situational": false,
```

Comment:
I think all SBAs should be considered situational.

Reply:
Basically dooming them to never get into logic considerations but maybe it's better that way for now.